layer/projects/ray_workflow_project_runner.py

```diff
@@ -63,6 +74,13 @@ def run(self) -> Run:
         ray.shutdown()
         return run
 
+    def _get_ray_proxy_url(self) -> str:
```

Comment:
Proxy URL is what I return from `getOrCreateCluster`. You don't need to add prefix or anything else to it I think.

Reply:
That bit would be changed to return the ray head node service name so that would be added to the header for proper routing. In this method we are building the URL for the ray proxy from the connected stack URL.